public/index.html

```diff
@@ -186,6 +186,150 @@ <h6 class="card-title text-center py-1" style="background-color: black;">Creator
         </div>
     </div>
 
+    <!-- card description--------------------------------------------------- -->
+
+    <div class="container bg-dark text-white">
+        <div class="px-3" id="createNewTask">
+            <div class="row pt-3">
+                <div class="col-md-11 col-10"></div>
+                <span type="button" class="col-md-1 col-2 mt-5 material-icons pb-3" id="backToMainFromCreateNewTask">
```

Comment:
Please convert all the margins into padding

Reply:
okay i will do